components/compliance-service/inspec-agent/scheduler/scheduler.go

```diff
@@ -93,15 +96,19 @@ func (a *Scheduler) PollForJobs(ctx context.Context) {
 // recurring jobs that are due to be run. For every due recurrent job,
 // we push that job onto the workflow queue.
 func (a *Scheduler) processDueJobs(ctx context.Context, nowTime time.Time) {
-	dueJobs := a.scanner.GetDueJobs(nowTime)
+	dueJobs, err := a.scanner.GetDueJobs(nowTime)
+	if err != nil {
+		logrus.Errorf("unable to retrieve due jobs: %s", err.Error())
+		return
+	}
 	if len(dueJobs) > 0 {
 		logrus.Debugf("processDueJobs, %d recurring jobs are due for running...", len(dueJobs))
 	}
 	for _, job := range dueJobs {
 		err := a.pushWorkflow(job, false)
 		if err != nil {
 			if err == cereal.ErrWorkflowInstanceExists {
-				logrus.Infof("Job %q is still/already running", job.Id)
+				logrus.Debugf("Job %q is still/already running", job.Id)
```

Comment:
Are we seeing this a lot? Seems like something we would want to keep at Info as it might mean compliance can't keep up with the schedule.  Ideally, we would surface this to the user in the UI

Reply:
ah ya that’s a good point. i haven’t seen it often, i was just over-aggressively changing info msgs to debug :) i’ll switch it back to info!

Reply:
ah ya that’s a good point. i haven’t seen it often, i was just over-aggressively changing info msgs to debug :) i’ll switch it back to info!